src/test/java/org/jboss/xavier/integrations/migrationanalytics/business/ManifestVersionServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package org.jboss.xavier.integrations.migrationanalytics.business;
```

Comment:
It seems this test is in the wrong package and it should be moved inside `versioning` to be consistent with `ManifestVersionService`'s package
```suggestion
package org.jboss.xavier.integrations.migrationanalytics.business.versioning;
```

Reply:
+1